cEP-0020.md

```diff
@@ -0,0 +1,590 @@
+# Newcomer metrics and Gamification
+
+| Metadata |                                                        |
+| -------- | ------------------------------------------------------ |
+| cEP      | 20                                                     |
+| Version  | 1.0                                                    |
+| Title    | Newcomer metrics and Gamification system               |
+| Authors  | Shrikrishna Singh <mailto:krishnasingh.ss30@gmail.com> |
+| Status   | Proposed                                               |
+| Type     | Process                                                |
+
+## Abstract
+
+This cEP describes a gamification system to automate the
+recognition of activities performed by people who join the
+coala community, and encourage them to progress towards
+obtaining the role of "Developer" in the community.
+
+This cEP is a part of the
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5892040252981248).
+
+For the implementation of this process, this project will
+incorporate a gamification system which will allow us to
+track the progress of newcomers and assign some points to
+each of the activities they do. There will be different
+levels based on point thresholds and a newcomer will also
+have some special privileges after the completion of each
```

Comment:
> have some special privileges

'special' is over-selling, especially when 'have' isnt true

use "be eligible for specific privileges"